lib/maze/configuration.rb

```diff
@@ -75,6 +75,9 @@ def initialize
     # Enables awareness of a public IP address on Buildkite with the Elastic CI Stack for AWS.
     attr_accessor :aws_public_ip
 
+    # Disables timestamp validation on spans
```

Comment:
Wording of the comment is potentially misleading, to me it makes it sound like setting it to true will disable it. 
```suggestion
    # Whether timestamp validation should be performed on spans
```